README.md

```diff
@@ -93,6 +93,10 @@ introduction of multiple architecture support. That one liner was no longer suff
 complex solution was needed to determine which architecture to download for. But if you're using
 linux, I think you can handle what needs to be done :-)*
 
+### Special Note regarding Ubuntu 22.04
+
+Ubuntu moved from libssl 1.1 to libssl 3.0 in 22.04. This currently breaks the updater. See https://github.com/rcdailey/trash-updater/issues/54
```

Comment:
There is a [Troubleshooting]() wiki page that I normally add this stuff to. I was planning on adding that before I saw your PR. Great minds think alike :-)

For the README.md, I was thinking of adding a "Requirements" section somewhere. Because in addition to the OpenSSL version, there's a minimum version requirement for Sonarr. Do you want to go ahead and move this to `wiki/Troubleshooting.md`? I'd hate to just close the PR and throw away your work.

Let me know if you'd prefer to do it. If so, I request you follow the existing bullet point format you already see there. Basically:

- Show the error message that a user might see (a small snippet of the stack trace you put in your open issue)
- Explain the reason for the error (you already did that in this PR, just cut & paste)
- Explain the solution (Maybe a small command line snippet showing the user needs to run `apt-get install libssl-dev` or whatever)
- Vanity links to existing github issues

Thanks!!

Reply:
I went ahead and added that Requirements section to README.md. Hopefully that helps. I won't do the Troubleshooting page until I hear back from you. Thanks again.

Reply:
Updated the request to reflect the changes you wanted.